bin/ch/WScriptJsrt.cpp

```diff
@@ -381,6 +382,14 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
 Error:
     if (errorCode != JsNoError)
     {
+        if (isInitializeThrow)
+        {
+            //clear unused exception
+            JsValueRef unusedException;
+            ChakraRTInterface::JsGetAndClearException(&unusedException);
```

Comment:
Could we just use `ChakraRTInterface::HasException` and clear based on that first?  That might be cleaner than having the macro set a local variable.

Reply:
make sense. will do.

---
In reply to: [212402959](https://github.com/Microsoft/ChakraCore/pull/5623#discussion_r212402959) [](ancestors = 212402959)